x-pack/metricbeat/modules.d/awsfargate.yml.disabled

```diff
@@ -1,5 +1,5 @@
 # Module: awsfargate
-# Docs: https://www.elastic.co/guide/en/beats/metricbeat/7.x/metricbeat-module-awsfargate.html
```

Comment:
@andresrc I wonder why we have a 7.x here as this is a patch release 🤔  Maybe something that was missed before?

Reply:
yep, we will need to review this